package.json

```diff
@@ -106,6 +108,7 @@
     "react-router-dom": "4.1.1",
     "react-test-renderer": "15.6.1",
     "rimraf": "2.6.1",
+    "string-loader": "0.0.1",
```

Comment:
Is the `string-loader` still being used?

Reply:
Nope. Removed.